### DISCUSSION TITLE
How to reset settings back to defaults using qalc CLI?

### DISCUSSION THREAD

**Discussion by cargocats** [Qalculate/libqalculate#565] [Q&A] [answered]
There is a setting for this on the GUI version of qalculate but I cant seem to find the setting for it on CLI version of qalculate.

**Comment by hanna-kn** [comment, marked as answer]
Either run `qalc --defaults` and enter command `save mode`, or delete qalc.cfg in the configuration folder (usually `~/.config/qalculate`, on GNU/Linux, or `c:\Users\[USERNAME]\AppData\Local\Qalculate`, on Windows).
